static/js/flow/profileTypes.js

```diff
@@ -22,12 +22,12 @@ export type DateEdit = {
 
 export type WorkHistoryEntry = {
   id?: ?number;
-  position?: ?string;
-  industry?: ?string;
-  company_name?: ?string;
-  start_date?: ?string|DateEdit;
-  start_date_edit?: ?string|DateEdit;
-  end_date?: ?string|DateEdit;
+  position: string;
```

Comment:
These all will still be `null` when a user first creates a new work history entry